packages/codec/lib/compiler/utils.ts

```diff
@@ -2,7 +2,7 @@ import debugModule from "debug";
 const debug = debugModule("codec:utils:compiler");
 
 import semver from "semver";
-import { CompilerVersion, SolidityFamily } from "@truffle/codec/types/compiler";
+import { CompilerVersion, SolidityFamily } from "./types";
 
 export function solidityFamily(compiler: CompilerVersion): SolidityFamily {
```

Comment:
Wasn't there a comment here before about the weird version string?  ...huh, I guess not.  Well, pretty sure I had it commented at some point.  Will have to add that comment back in.